src/main/resources/assets/wurst/lang/zh_cn.json

```diff
@@ -1,139 +1,139 @@
-  "description.wurst.hack.jesus": "允许你走在水面上.\n耶稣曾经用这个功能 约~2000 年前.",
-  "description.wurst.hack.jetpack": "模仿你好像真的有飞行背包一样飞行.\n\n§c§l警告:§r 如果你不使用 NoFall ,会导致受到掉落伤害..",
-  "description.wurst.hack.kaboom": "像爆炸一样破坏你附近的方块.\n这个可能比 Nuker 功能破坏更快如果服务器没有反作弊的话. 如果使用适合的工具和破坏脆弱的方块的效果会更好.\n注意: 这并不是真正的爆炸.",
-  "description.wurst.hack.killauralegit": "KillauraLegit 使得更难检测.\n在正常的NoCheat+(NCP)服务器上不需要!",
-  "description.wurst.hack.killaura": "自动攻击你附近的实体.",
-  "description.wurst.hack.killpotion": "生成一个能杀死绝大多数生物的药水,包括创造模式的玩家.但对那些不死的生物或者已死生物无效.\n\n需要在创造模式下.",
-  "description.wurst.hack.liquids": "允许你在水中直接放方块.",
-  "description.wurst.hack.lsd": "产生吃了云南的蘑菇一样的效果.",
-  "description.wurst.hack.masstpa": "给每个人发送tpa请求.\n如果有人接了就停下来.",
-  "description.wurst.hack.mileycyrus": "就是类似疯狂按Shift潜行.",
+  "description.wurst.hack.anchoraura": "自动放置（可选）并引爆重生锚，杀死你附近的生物。",
+  "description.wurst.hack.antiafk": "随机走动，用于避免服务器挂机检测。\n需要至少有 3x3 的区域可以走动。",
+  "description.wurst.hack.antiblind": "防止失明。不兼容 OptiFine。",
+  "description.wurst.hack.anticactus": "保护你免受仙人掌的伤害。",
+  "description.wurst.hack.antiknockback": "让你不会被其他生物推动或击退。",
+  "description.wurst.hack.antispam": "预防刷屏，用计数器显示重复内容。",
+  "description.wurst.hack.antiwaterpush": "防止你被水流推动。",
+  "description.wurst.hack.antiwobble": "关闭因反胃和传送门引起的摇晃效果。",
+  "description.wurst.hack.autoarmor": "自动管理你的盔甲。",
+  "description.wurst.hack.autobuild": "自动建筑。\n放置一个方块以开始建造。",
+  "description.wurst.hack.autodrop": "自动丢弃你不想要的物品。",
+  "description.wurst.hack.autoleave": "当生命值过低时，自动退出服务器。",
+  "description.wurst.hack.autoeat": "让你自动进食。",
+  "description.wurst.hack.autofarm": "自动收获或种植作物。\n有效作物：小麦、胡萝卜、马铃薯、甜菜根、南瓜、西瓜、仙人掌、甘蔗、海带、竹子、地狱疣、可可豆。",
+  "description.wurst.hack.autofish": "自动使用最好的鱼竿钓鱼。",
+  "description.wurst.hack.automine": "自动挖掘你眼前的方块。",
+  "description.wurst.hack.autopotion": "当生命值过低时，自动扔一瓶喷溅型的瞬间恢复药水。",
+  "description.wurst.hack.autoreconnect": "当你被服务器踢出时自动重连。",
+  "description.wurst.hack.autorespawn": "死亡时自动重生。",
+  "description.wurst.hack.autosign": "立即在每个告示牌上写下任何你想要的文字。\n当激活之后，需要在放置的第一个牌子上写你想要的文字，\n之后接下来的所有牌子都将自动填写一样的文字。",
+  "description.wurst.hack.autosoup": "当生命值过低时，自动喝汤。\n\n§l注意：§r 这个功能忽视饥饿值，因为在某些服务器的小游戏中，喝汤直接补充生命值，如果你正玩的服务器没有这个功能，请使用 AutoEat。",
+  "description.wurst.hack.autosprint": "让你自动疾跑。",
+  "description.wurst.hack.autosteal": "自动从你打开的箱子中偷窃所有物品。",
+  "description.wurst.hack.autoswim": "在水中自动切换为泳姿。",
+  "description.wurst.hack.autoswitch": "自动在物品栏中快速切换。\n\n§l提示：§r 与 BuildRandom 搭配使用，效果更佳。",
+  "description.wurst.hack.autosword": "当你攻击生物的时候，自动使用快捷栏中最好的剑。\n提示: 建议与 KillAura 搭配。",
+  "description.wurst.hack.autotool": "自动切换成物品栏中最优的破坏工具。",
+  "description.wurst.hack.autototem": "当你即将死亡时，自动将图腾移动到副手上。",
+  "description.wurst.hack.autowalk": "让你自动走路。",
+  "description.wurst.hack.basefinder": "通过寻找人为放置的方块找到玩家的基地。\n假如这种方块被找到将会被高亮。",
+  "description.wurst.hack.blink": "暂停所有动作更新，并在关闭时继续。",
+  "description.wurst.hack.boatfly": "允许你使用船飞天。",
+  "description.wurst.hack.bonemealaura": "在指定的作物上自动使用骨粉。",
+  "description.wurst.hack.bowaimbot": "使用弓弩时自动瞄准。",

```

Comment:
Am I reading this wrong or does this translation no longer mention bows? BowAimbot works for both bows and crossbows and the description should say so.

Reply:
This does mention bows, it means "bows and crossbows".